### ISSUE TITLE
Info on model

### ISSUE THREAD

**Issue by matteosodano** [jan-ackermann/Maskomaly#5]
Hi guys and thanks for your nice work!
I was just wondering whether you can provide two things about your model: number of parameters and either MACs or GFLOPs (for this, I would advise to `pip install torchprofile` and then `from torchprofile import profile_macs; macs = profile_macs(linear_model, sample_data)`, unless you have a better internal solution. Let me know!
Thank you so much!

**Comment by jan-ackermann** [comment]
We use Mask2Former with SWIN-L backbone trained on Cityscapes for semantic segmentation. We do not train anything. So our parameters should be: 215455112

I am quite busy right now and can't check the FLOPs. Based on some research the Mask2former does 937 GFLOPs.
Our method on top does not have a fixed count (it depends on the Mask2Former logits).
Based on calculations, in normal scenarios our method would do significantly less than 10 GFLOP on top.

A vectorized implementation of our method(without the borders) averages less than 0.5 ms on a 4070.

I hope this helps you.

**Comment by matteosodano** [comment]
It does help. Thank you very much!
